.jscsrc

```diff
@@ -1,7 +1,7 @@
 {
 	"preset": "jquery",
 	"requireCamelCaseOrUpperCaseIdentifiers": "ignoreProperties",
-	"maximumLineLength": 250,
+	"maximumLineLength": 300,
```

Comment:
The new namespaces are longer, making jscs throw a fit because of many lines over 250 chars. I'd like to replace jscs with eslint at some point, so I've temporarily increased this setting.